build.gradle.kts

```diff
@@ -37,6 +42,23 @@ dependencies {
 }
 
 tasks {
+    val fatJar = getByName<ShadowJar>("shadowJar") {
+        manifest {
+            attributes["Main-Class"] = "com.cjbooms.fabrikt.cli.CodeGen"
+            attributes["Implementation-Title"] = "fabrikt"
+            attributes["Implementation-Version"] = project.version
```

Comment:
Not sure how you want to manage the versioning @cjbooms

Reply:
We should go semver for the public release I guess

Reply:
I guess we can start at 1.0 saying as this is a port of a successful internal project